### PR TITLE
FPLA-1325 config for adoption-backend

### PR DESCRIPTION
**Pull request by Gogetter**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by Gogetter** [comment]
@akomisarek @tomaszpowroznik pr as discussed.

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: adoption-backend
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: adoption-backend
+    version: 1.3.0
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: This should match https://github.com/hmcts/adoption-backend/blob/master/charts/adoption-backend/Chart.yaml#L6

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  values:
+    java:
+      replicas: 2
+      readinessDelay: 45
+      readinessTimeout: 5
+      readinessPeriod: 15
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: Are these intentional ?

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: 4553bfbf-2d36-4331-adf1-97d645a83189
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: Better move it to secrets

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      image: hmctspublic.azurecr.io/adoption/backend:prod-32fcbe58
+      ingressHost: adoption-backend-aat.service.core-compute-aat.internal
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: Better move it as templated to values.yaml

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+      environment:
+        IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
+        IDAM_S2S_AUTH_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+        CORE_CASE_DATA_API_URL: http://ccd-data-store-api-aat.service.core-compute-aat.internal
+        IDAM_CLIENT_ID: adoption_backend
+        IDAM_CLIENT_REDIRECT_URI: https://adoption-backend-*.service.core-compute-aat.internal/oauth2/callback
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.aat.platform.hmcts.net
+        AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+      aadIdentityName: adoption
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: All these should move to base chart

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+      keyVaults:
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: Not needed if you don't need secrets

File: k8s/aat/common/adoption/adoption-backend.yaml [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        adoption:
+          resourceGroup: adoption
+          secrets:
+      prometheus:
+        enabled: true
+    global:
+      environment: aat
+      subscriptionId: "1c4f0704-a29e-403d-b719-b90c34ef14c9"
```
Comment (adusumillipraveen) [inline k8s/aat/common/adoption/adoption-backend.yaml]: Not needed with latest charts

Comment (Gogetter) [reply k8s/aat/common/adoption/adoption-backend.yaml]: updated

**Comment by Gogetter** [comment]
@adusumillipraveen all updated now. Also did a pr [here](https://github.com/hmcts/adoption-backend/pull/15)

File: k8s/aat/common/adoption/adoption-backend.yaml (hunk 2 above)
Comment (Gogetter) [reply k8s/aat/common/adoption/adoption-backend.yaml]: updated as discussed

**Pull request merged**